syntax = "proto3";
package isegram;

service PostsService {
  // Sends a greeting
  rpc GetPosts (PostsRequest) returns (PostsReply) {}
}

message Post {
  int32 id = 1;
  int32 user_id = 2;
  string text = 3;
}

message PostsRequest {
}

message PostsReply {
  repeated Post posts = 1;
}